doc/release/release_0.14.rst

```diff
@@ -1,3 +1,75 @@
+Announcement: scikit-image 0.14.3
+=================================
+
+This is a bugfix release, and contains the following changes from v0.14.2:
+
+API Changes
+-----------
+-  morphology.local_maxima now returns a boolean array instead of uint8 (#3749,
+   #3752)
+
+
+Bug Fixes
+---------
+- _marching_cubes_lewiner_cy: mark char as signed (#3587, #3678)
+- Fix potential use of NULL pointer (#3696)
+- pypi: explicitly exclude Python 3.1, 3.2, and 3.3 (#3726)
+- Reduce default tolerance in threshold_li (#3622) (#3781)
+- Denoise now accepts float32 datatypes (#3449) (#3486) (#3880)
+
+Other Pull Requests
+-------------------
+
+- BLD: pin cython's language_level (#3716)
```

Comment:
```suggestion
- BLD: pin cython's language_level to 3 to ensure compatibility with future versions of cython (#3716)
```

Reply:
This is an implementation detail that I don't think needs to be in the release notes.